cluster/src/table_manager.rs

```diff
@@ -52,6 +52,14 @@ impl TableManager {
         Ok(())
     }
 
+    pub fn contains_shard(&self, shard_id: ShardId) -> bool {
+        self.inner
+            .read()
+            .unwrap()
+            .shard_infos
+            .contains_key(&shard_id)
+    }
+
     pub fn update_table_info(&self, shard_table: &HashMap<ShardId, ShardTables>) {
```

Comment:
I think the name ShardTables is too strange, how about TablesInShard?
And rename shard_table to shard_to_tables?

Reply:
There is no significant difference between `TablesInShard` and `ShardTables`, but it is hard to change such a name because the protobuf files should be changed also. So I prefer to keep this name.

As for the parameter name `shard_table`, I will replace it with `shard_tables`.

Reply:
When I see shard table, I will think it is a table used to record the information of shard.